src/cowpatch/base_elements.py

```diff
@@ -220,6 +231,39 @@ def __mul__(self, other):
     def __and__(self, other):
         raise ValueError("currently not implimented &")
 
+    def _get_grob_tag_ordering(self):
```

Comment:
in `patch` object as it interacts with the `layout` and the `annotation` tag structure